setup.py

```diff
@@ -6,7 +6,7 @@
     author='MotiveMetrics',
     install_requires=[
         'beanstalkc3>=0.4.0',
-        'couchbase==2.5.12',
+        'couchbase>=3.0',
```

Comment:
Are you sure you don't want to lock down the version?

Reply:
I usually try not to, because they get stagnant and then we eventually end up with a giant project (like this one) to get back up to date.

It has generally worked for me because I can fix things pretty quickly when they break. As we start bring on new engineers and distribute responsibility more that may not be true any more.

Let's leave it as-is for now, but we'll also keep your thought in mind.